sbol_utilities/__init__.py

```diff
@@ -0,0 +1 @@
+from .excel_to_sbol import excel_to_sbol
```

Comment:
This line is not necessary at this point.  Even without this line, you can still import it as `from sbol_utilities.excel_to_sbol import excel_to_sbol`.  The reason that I lean toward this latter pattern (although it appears a little redundant) is because I anticipate that each utility will eventually evolve to have multiple methods, and it will be nice to have these methods scoped to a particular utility namespace rather than cluttering the global namespace.

Reply:
Rather than the `from ... import` pattern, I've been trying to keep my global namespace clean by doing `import package` and then invoking with `package.function`. With that pattern, I was having to do `sbol_utilities.excel_to_sbol.excel_to_sbol`, which feels awkward.

Reply:
The situation I'm trying to avoid (and what I mean by cluttering the namespace) is this:
```
sbol_utilities.utilityA_method1
sbol_utilities.utilityA_method2
sbol_utilities.utilityB_method1
sbol_utilities.utilityB_method2
sbol_utilities.utilityC_method1
sbol_utilities.utilityC_method2
```
In this case, if I import `sbol_utilities` I get all of the methods jumbled together in the package namespace.  Whereas, what I'm advocating is a subpackage organization like this:
```
sbol_utilities.utilityA.method1
sbol_utilities.utilityA.method2
sbol_utilities.utilityB.method1
sbol_utilities.utilityB.method2
sbol_utilities.utilityC.method1
sbol_utilities.utilityC.method2
```
Now, if I `import sbol_utilities.utilityA`, I only import those methods associated with utilityA.

A less verbose solution for your example above is to import the subpackage as follows:
```
import sbol_utilities.excel_to_sbol as excel_to_sbol
excel_to_sbol.excel_to_sbol(arg0, arg1, ...)
```
While this may still seem a little redundant and silly at the moment, it will appear less so as the utilities expand to encompass more methods, e.g.:
```
excel_to_sbol.sbol_to_excel
excel_to_sbol.validate_spreadsheet
excel_to_sbol.merge_spreadsheets
```